doc/Developers.md

```diff
@@ -3,22 +3,34 @@ Developer Environment
 
 Sia is written in golang. To build and test Sia, you are going to need a
 working go environment, including having both $GOROOT/bin and $GOPATH/bin in
-your $PATH. For most Linux distributions, go will be in the package manager.
-Then it should be sufficient to run `make dependencies && make`. For more
-information, check the [go documentation](http://golang.org/doc/install).
-
-If you plan on cross compiling Sia, you may need to install go from source. You
-can find information on that [here](http://golang.org/doc/install/source).
-
-When you clone the Sia repository, make sure that you call `git clone` from the
-folder `$GOPATH/src/github.com/NebulousLabs/`. It will not be sufficient to clone
-Sia from the gopath. Alternatively, you should be able to call `go get -u
-github.com/NebulousLabs/Sia/...`.
-
-Golang does not like sym links, a problem which seems to have appeared
-recently. If you are using a sym link, you can expect to see an error similar
-to the one below:
-
+your $PATH. For most Linux distributions, go will be in the package manager,
+though it may be an old version that is incompatible with Sia. Once you have a
+working go environment, you are set to build the project. If you plan on cross
+compiling Sia, you may need to install go from source. You can find information
+on that [here](http://golang.org/doc/install/source).
+
+Sia has has a development build, an automated testing build, and a release
+build. The release build is the only one that can synchronize to the full
+network. To get the release build, it is usually sufficient to run
+`go get -u github.com/NebulousLabs/Sia/...`. This will pull all of the
+necessary repositories off of github and place them in the correct folders
+within the gopath.
+
+If you would like to build the project manually, you will need to clone the Sia
+repository into the folder `$GOPATH/src/github.com/NebulousLabs/`. It will not
+be sufficient to close Sia from the gopath. It also seems that more recent
+versions of go do not like sym links. After cloning, you will need to run 
+`make dependencies`.
```

Comment:
I'm 95% sure that if you use symlinks to put the github repo in your gopath, go will be unable to find the packages it needs